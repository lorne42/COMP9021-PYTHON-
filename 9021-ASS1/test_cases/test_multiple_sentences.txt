Last day, I met Sirs Ian and Jason.
Sir Ian said: "At least one of us is a Knave."
Sir Jason glanced at him, and murmured: "I 
am a Knight." Sir Ian retorted: "Sir Jason is a Knave."