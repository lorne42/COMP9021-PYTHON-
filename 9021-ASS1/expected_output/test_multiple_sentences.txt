The Sirs are: Ian Jason
There is a unique solution:
Sir Ian is a Knight.
Sir Jason is a Knave.